test/powershell/Modules/Microsoft.PowerShell.Management/Get-Item.Tests.ps1

```diff
@@ -191,15 +191,17 @@ Describe "Get-Item environment provider on Windows with accidental case-variant
         if (-not (Get-Command -ErrorAction Ignore node.exe)) {
             Write-Warning "Test skipped, because prerequisite Node.js is not installed."
         } else {
+            $fso = New-Object -ComObject Scripting.FileSystemObject
+            $shortPath = $fso.GetFile("$PSHOME\pwsh.exe").ShortPath.Replace('\', '/')
             $valDirect, $valGetItem, $unused = node.exe -pe @"
                 env = {}
                 env.testVar = process.env.testVar // include the original case variant with its original value.
                 env.TESTVAR = 'b' // redefine with a case variant name and different value
                 // Note: Which value will win is not deterministic(!); what matters, however, is that both
-                //       $env:testvar and Get-Item env:testvar report the same value.
+                //       `$env:testvar and Get-Item env:testvar report the same value.
                 //       The nondeterministic behavior makes it hard to prove that the values are *always* the
                 //       same, however.
-                require('child_process').execSync(\"\\\"$($PSHOME -replace '\\', '/')/pwsh.exe\\\" -noprofile -command `$env:testvar, (Get-Item env:testvar).Value\", { env: env }).toString()
+                require('child_process').execSync("$shortPath -noprofile -command `$env:testvar, (Get-Item env:testvar).Value", { env: env }).toString()
```

Comment:
How could we check that it is not fail as previously? `Should -Not -Throw` or check `$?`?

Reply:
Should probably check that $LastExitCode is 0 I think.